### ISSUE TITLE
Feature: Appropriate page titles for different pages

### ISSUE THREAD

**Issue by Abhinav-26** [devtron-labs/devtron#2356]
### 🔖 Feature description

In Devtron, if you navigate to different pages say - `App Details`, `Build & Deploy`, `App Configurations`, etc, all the page titles are shown as `DevTron Labs` in the browser. It should show the page name at which you are navigating to in runtime. 

For eg - 

Here we are at `App Details` tab but still title is shown as `DevTron Labs`

<img width="1440" alt="App Details" src="https://user-images.githubusercontent.com/48083659/190078864-494725af-64a2-4043-95d6-dad2debf70d1.png">

and again in this, we are at `App Configurations` tab but still the title is `DevTron Labs`

<img width="1440" alt="App Configurations" src="https://user-images.githubusercontent.com/48083659/190079124-6f892e6e-0a3a-4417-907c-3a8204abd9c5.png">

### 🎤 Pitch

n/a

### ✌️ Solution

n/a

### 🔄️ Alternative

n/a

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/devtron-labs/devtron/blob/main/CODE_OF_CONDUCT.md)

**Comment by abhiyant-10** [comment]
I just started reading about devtron I would like to take up this issue if it still persists

**Comment by Kanika637** [comment]
May I work on this issue @Abhinav-26 ?

**Comment by rakshitgondwal** [comment]
Can I work on this @Abhinav-26 ?

**Comment by t-trafer** [comment]
hi @Abhinav-26, i can work on this issue

**Comment by Abhinav-26** [comment]
> hi @Abhinav-26, i can work on this issue

Hello @t-trafer , yes please go ahead. You can start working on this and if you have any questions or doubts, feel free to join our [discord community](https://discord.devtron.ai) and ask your questions in `#contributions`

**Comment by barkhaaroraa** [comment]
Hi could I work on this?

**Comment by t-trafer** [comment]
> Hello @t-trafer , are you working on this issue??

no, unable to set up the project - you can assign this to @barkhaaroraa

**Comment by Abhinav-26** [comment]
Please go ahead with this @barkhaaroraa
